src/ansible_creator/resources/ansible_project/collections/requirements.yml.j2

```diff
@@ -0,0 +1,17 @@
+---
+collections:
+  - name: ansible.builtin
```

Comment:
This isn't really a collection by itself. It's just a FQCN convention for using core modules in a task.

Reply:
I would recommend adding these at least (and remove `community.general` for now):
```yaml
  - name: ansible.posix
  - name: ansible.scm
  - name: ansible.utils
```